src/Common/Models/SensorModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Middleware.Common.Models
+{
+    public class SensorModel
+    {
+        public string SensorName { get; set; }
+        public string SensorType { get; set; }
+        public string SensorLocation  { get; set; }
+        public string SensorDescription { get; set; }
+        public List<RosTopicModel> RosTopicPub { get; set; } //A sensor can publish multiple topics
```

Comment:
Same as before. I would suggest to have following property names:
* Name 
* Type
* Location
* Description
* PublishedRosTopics
In the context of this model we know that we are talking about the sensor :)

Reply:
done :)